docs/TypeScript.md

```diff
@@ -2,11 +2,11 @@
 
 <a id="typescript"></a>
 ## TypeScript
-Fastify is shipped with a typings file, but it still require to install `@types/node`, depending on the Node.js version that you are using.
+Fastify is shipped with a typings file, but you may need to install `@types/node`, depending on the Node.js version you are using.
 
 ## Types support
-We do care about the TypeScript community, but the framework is written in plain JavaScript and currently no one of the core team is a TypeScript user while only one of the collaborators is.
-We do our best to have the typing updated with the latest version of the API, but *it can happen* that the typings are not in sync.<br/>
+We do care about the TypeScript community, but the framework is written in plain JavaScript and currently no one on the core team is a TypeScript user while only one of the collaborators is.
+We do our best to have the typings updated with the latest version of the API, but *it can happen* that the typings are not in sync.<br/>
```

Comment:
This is not true anymore, we have multiple typescript users in the core team. @EthanArrowood is currently doing most of the work on typescript, and he is reworking all our types.

@fastify/core I propose we change this, removing the sentence about not having typescript core maintainers. Wdyt?

Reply:
I'm also learning it in order to check simples PR

For what I saw, I would remove this sentence in next months

Reply:
> I propose we change this, removing the sentence about not having typescript core maintainers. Wdyt?

👍

Reply:
I agree!

Reply:
I'm glad I was able to spur such a productive discussion.

Reply:
Thanks! We needed this discussion, and updating this text, considering the state of things. That’s the benefit of open governance.

Would you mind updating that text to reflect this discussion?

Reply:
Done. It's in commit 4321eb2.